packages/components/src/components/Alert/Alert.tsx

```diff
@@ -173,36 +173,19 @@ export const Alert: FC<AlertProps> = ({
 
   const iconDisplay = (
     <View style={iconViewStyle}>
-      <Icon fill={contentColor} {...iconProps} preventScaling />
+      <Icon fill={contentColor} name={iconName} preventScaling />
       <Spacer horizontal />
     </View>
   )
 
-  const expandIconProps: IconProps = {
-    fill: contentColor,
-    width: Sizing._16,
-    height: Sizing._16,
-    maxWidth: Sizing._24,
-    name: expanded ? 'ChevronUp' : 'ChevronDown',
-  }
-
   const expandableIcon = (
     <View style={iconViewStyle}>
       <Spacer horizontal />
-      <Icon {...expandIconProps} />
-    </View>
-  )
-
-  /**
-   * When an alert is expandable, the content should have additional padding on
-   * the right to appear within the expandable icon. Since the expandable icon
-   * has a maxWidth, this hidden icon matches the spacing of the icon insteading
-   * instead of adding a <Spacer /> with a calculated value.
-   */
-  const spacerIcon = (
-    <View style={iconViewStyle} aria-hidden>
-      <Spacer horizontal />
-      <Icon {...expandIconProps} fill="none" />
```

Comment:
Can you fill me in on why this is no longer needed with the new Icon?

Reply:
Because we'd added it to account for the height/width being 16 and maxWidth 24 so it would scale with font scaling. 

Now the icon is set at height/width 24 always so it no longer scales and can just be hard coded to 24 (done below with Spacer of 10 (padding to expand/collapse icon) + 24 (expand/collapse icon).

Reply:
Here's [the Slack discussion around changing it to be 24x24](https://adhoc.slack.com/archives/C05DQPHUVNZ/p1717090789699079) always--as the SVG path off the new expand/collapse icon is much smaller than previously.